autogpt/logs.py

```diff
@@ -46,15 +46,19 @@ def __init__(self):
         self.console_handler.setFormatter(console_formatter)
 
         # Info handler in activity.log
-        self.file_handler = logging.FileHandler(os.path.join(log_dir, log_file))
+        self.file_handler = logging.FileHandler(
+            os.path.join(log_dir, log_file), 'a', 'utf-8'
+        )
         self.file_handler.setLevel(logging.DEBUG)
         info_formatter = AutoGptFormatter(
             "%(asctime)s %(levelname)s %(title)s %(message_no_color)s"
         )
         self.file_handler.setFormatter(info_formatter)
 
         # Error handler error.log
-        error_handler = logging.FileHandler(os.path.join(log_dir, error_file))
+        error_handler = logging.FileHandler(
+            os.path.join(log_dir, error_file), 'a', 'utf-8'
```

Comment:
Specifying the mode `'a'` is unnecessary, since this is the default. Just adding `encoding='utf-8'` is enough, and is also more readable.

So the change becomes:

```
error_handler = logging.FileHandler(os.path.join(log_dir, error_file), encoding='utf-8')
```